gke-policies/policy/network_policies_test.rego

```diff
@@ -0,0 +1,35 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+package gke.policy.network_policies_engine
+
+test_dataplane_v1_without_netpol {
+  valid with input as {"name": "test-cluster",  "addons_config": { "network_policy_config": {"disabled": true }},"private_cluster_config": {"enable_private_nodes": true}}
```

Comment:
Shouldn't this test the **not** valid?
In this example, the cluster has network policies disabled.

Reply:
@rickygodoy you're totally right here, there's something strange. Let me do an entire re-check